spec/schemas/releases_spec.rb

```diff
@@ -6,7 +6,7 @@ module Bosh::Workspace::Schemas
 
     subject { Releases.new.validate(releases) }
 
-    %w(name version git).each do |field_name|
+    %w(name version).each do |field_name|
```

Comment:
Could you add `git` to the test for the [optional attributes](https://github.com/wayneeseguin/bosh-workspace/blob/gitoptional/spec/schemas/releases_spec.rb#L26-L29)